cg/models/run_devices/pacbio_cell_directory_data.py

```diff
@@ -0,0 +1,20 @@
+from pathlib import Path
+
+from cg.constants.pacbio import PacBioDirsAndFiles
+from cg.utils.files import get_files_in_directory_with_pattern
+
+
+class PacBioCellDirectoryData:
```

Comment:
I find this naming confusing, it sounds like the class will encapsulate data but it is just paths

Reply:
removed